asab/library/providers/zookeeper.py

```diff
@@ -1,132 +1,124 @@
 import logging
+import functools
 import os.path
 import urllib.parse
-import asab.zookeeper
-import yaml
+
+import kazoo.exceptions
 
 from .abc import LibraryProviderABC
+from ..item import LibraryItem
+from ...zookeeper import ZooKeeperContainer
 
 #
 
 L = logging.getLogger(__name__)
 
-
 #
 
 
 class ZooKeeperLibraryProvider(LibraryProviderABC):
 
-	def __init__(self, app, path):
-		super().__init__(app, path)
-		self.App = app
-		self.Path = path
-		url_pieces = urllib.parse.urlparse(self.Path)
+	def __init__(self, library, path):
+		super().__init__(library)
+
+		url_pieces = urllib.parse.urlparse(path)
```

Comment:
I guess you try to do this line of code twice - once here and once in the kazoo wrapper